components/authz-service/server/v2/rules_test.go

```diff
@@ -470,6 +471,7 @@ func TestGetRule(t *testing.T) {
 				Type:       api.ProjectRuleTypes_NODE,
 				ProjectId:  projectID,
 				Conditions: apiConditions,
+				Status:     "applied",
```

Comment:
Maybe this could be a constant? We have some at the DB level after the `DeleteRule` refactor PR merged.

Reply:
yeah a constant in a different package :wink: this is only tests I'll add another package-level one.